wordpress-mobile/WordPress-iOS
# Swift App Delegate 3: Third Party SDK configuration

Continuing from #8258, this PR moves methods for configuring the various third party SDKs we use over to Swift. Ported over:

* Reachability
* Hockey
* Analytics
* Crashlytics
* App Rating Utility

I also removed some unnecessary StoreKit code, and fixed a logic issue with analytics where it didn't reflect the way the editor and NUX are presented these days.

As with the previous PRs for the Great Swiftening, this PR is primarily concerned with just getting the code translated to Swift as-is – we can make improvements later.

To test:

* Check the Swift code against the obj-c code
* Ensure the app builds correctly for both Debug and Release builds (as there's some conditional code in there) and that tests pass
